BackofficeBundle/Resources/public/coffee/UserFormView.coffee

```diff
@@ -0,0 +1,26 @@
+###*
+ * @namespace OpenOrchestra
+###
+window.OpenOrchestra or= {}
+
+###*
+ * @class UserFormView
+###
+class OpenOrchestra.UserFormView extends FullPageFormView
+
+  ###*
+   * On view ready
+  ###
+  onViewReady: ->
+    success = $('.alert-success', @$el).length > 0
+    if @options.submitted && success
+      entityId = $('#oo_user', @$el).first().data('user-id')
```

Comment:
may be use the same mecanic than other entities : add an hidden input which contains the userId, you have an example you the templateType